modules/openapi-generator/src/test/java/org/openapitools/codegen/java/AbstractJavaCodegenExampleValuesTest.java

```diff
@@ -166,4 +166,52 @@ void customExampleForEnumValue() {
             Assert.assertEquals(p.example, "CustomEnumValue");
         }
     }
+    @Test
+    void customExampleForEnumValue() {
+        final AbstractJavaCodegen fakeJavaCodegen = new P_AbstractJavaCodegen();
+        final CodegenParameter p = new CodegenParameter();
+        p.allowableValues = Collections.singletonMap("values", Arrays.asList("first", "second"));
+        p.dataType = "WrappedEnum";
+        p.example = "CustomEnumValue";
+
+        fakeJavaCodegen.setParameterExampleValue(p);
+        Assert.assertEquals("CustomEnumValue", "CustomEnumValue");
```

Comment:
The assertion in this test and the test below doesn't test anything useful.

Reply:
@martin-mfg  I have removed the unnecessary  test and assertions